### PR TITLE
added gohugo.io plugin

### PR DESCRIPTION
**Pull request by mgladdish**

### CODE REVIEW

**Comment by Stratus3D** [comment]
Actually, you do want to list `jq` as a dependency in your readme? Users of your plugin won't get very far without it.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ place for people (and asdf itself) to look for plugins.
 | Elm       | [vic/asdf-elm](https://github.com/vic/asdf-elm) | [![Build Status](https://travis-ci.org/vic/asdf-elm.svg?branch=master)](https://travis-ci.org/vic/asdf-elm)
 | Erlang    | [asdf-vm/asdf-erlang](https://github.com/asdf-vm/asdf-erlang) | [![Build Status](https://travis-ci.org/asdf-vm/asdf-erlang.svg?branch=master)](https://travis-ci.org/asdf-vm/asdf-erlang)
 | Go        | [kennyp/asdf-golang](https://github.com/kennyp/asdf-golang) | [![Build Status](https://travis-ci.org/kennyp/asdf-golang.svg?branch=master)](https://travis-ci.org/kennyp/asdf-golang)
+| GoHugo    | [mgladdish/asdf-gohugo](https://bitbucket.org/mgladdish/asdf-gohugo) |
```
[the remark this answers is not in the document]
Comment (mgladdish) [reply README.md]: Ah, it turns out there is. Via unsupported third parties, but it should be OK.

[img]https://bitbucket-badges.atlassian.io/badge/mgladdish/asdf-gohugo.svg?ref=master[/img
]

File: plugins/gohugo [hunk 2]
```diff
@@ -0,0 +1 @@
+repository = https://mgladdish@bitbucket.org/mgladdish/asdf-gohugo.git
```
[the remark this answers is not in the document]
Comment (mgladdish) [reply plugins/gohugo]: You're quite right. Will fix.

**Comment by Stratus3D** [comment]
Thanks!

**Pull request merged**